daemon/cmd/daemon_main.go

```diff
@@ -1193,8 +1183,12 @@ func initDaemonConfig(vp *viper.Viper) {
 		neighborsmap.SizeofNeighKey6+neighborsmap.SizeOfNeighValue,
 		lbmap.SizeofSockRevNat6Key+lbmap.SizeofSockRevNat6Value)
 
+	option.Config.SetupLogging(vp, "cilium-agent")
 	option.Config.Populate(vp)
```

Comment:
I think it would be useful to mention in the `Populate()` godoc that it depends on `SetupLogging()` being called first.

Reply:
It's not hard depending on, such as in unit tests. Will add a note regardless.